code/modules/mob/living/simple_animal/abnormality/he/puss_in_boots.dm

```diff
@@ -0,0 +1,325 @@
+		forceMove(jump_turf)
+		if(ishuman(target))
+			var/mob/living/carbon/human/H = target
+			H.Stun(9)
+		else
+			can_act = TRUE
+		SLEEP_CHECK_DEATH(3)
+		playsound(target, 'sound/abnormalities/pussinboots/counterslash.ogg', 50, 0, 2)
+		SLEEP_CHECK_DEATH(6)
+		playsound(target, 'sound/abnormalities/crumbling/attack.ogg', 50, 1)
+		Finisher(target)
+	icon_state = icon_aggro
+	can_act = TRUE
+	finishing = FALSE
+
+/mob/living/simple_animal/hostile/abnormality/puss_in_boots/proc/Finisher(mob/living/target)
+	target.apply_damage(50, PALE_DAMAGE, null, target.run_armor_check(null, RED_DAMAGE)) //50% of your health in red damage
+	to_chat(target,"<span class='danger'>[src] is trying to cut you in half!</span>")
+	if(ishuman(target))
+		if(target.health < 0)
+			var/mob/living/carbon/human/H = target
+			H.gib() //maybe we can get a special gib effect for cutting someone in half someday.
+
+/mob/living/simple_animal/hostile/abnormality/puss_in_boots/OpenFire()
+	if(!can_act)
+		return
+	if((finisher_cooldown < world.time) && prob(50))
+		Execute(target)
+	return
+
+//Death/Defeat
+/mob/living/simple_animal/hostile/abnormality/puss_in_boots/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 5 SECONDS) //for some reason he just disappears
+	QDEL_IN(src, 5 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/puss_in_boots/proc/escape()
+	if(finishing) //we dont wanna interrupt
+		addtimer(CALLBACK(src, .proc/escape), 7 SECONDS)
+		return
+	density = FALSE
+	animate(src, alpha = 0, time = 5 SECONDS)
+	QDEL_IN(src, 5 SECONDS)
+
+//Status Effects
+/datum/status_effect/chosen
+	id = "chosen"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	alert_type = null
+	var/attribute_bonus = 0
+	tick_interval = 600 //stats update every 60s
+	var/obj/item/ego_weapon/lance/chosen_arms = null
+
+/datum/status_effect/chosen/on_apply()
+	if(!ishuman(owner))
+		return
+	var/mob/living/carbon/human/user = owner
+	to_chat(user, "<span class='nicegreen'>You feel protected.</span>")
+	user.add_overlay(mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "inheritance", -MUTATIONS_LAYER))
+	StatUpdate(user)
+	user.physiology.red_mod *= 0.2
+	user.physiology.white_mod *= 0.2
+	user.physiology.black_mod *= 0.2
+	user.physiology.pale_mod *= 0.2
+	chosen_arms = new /obj/item/ego_weapon/lance/famiglia(get_turf(user))
+	return ..()
+
+/datum/status_effect/chosen/on_remove()
+	if(!ishuman(owner))
+		return
+	var/mob/living/carbon/human/user = owner
+	user.cut_overlay(mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "inheritance", -MUTATIONS_LAYER))
+	user.physiology.red_mod /= 0.3
+	user.physiology.white_mod /= 0.3
+	user.physiology.black_mod /= 0.3
+	user.physiology.pale_mod /= 0.3
+	for(var/attribute in user.attributes)
+		user.adjust_attribute_buff(attribute, -attribute_bonus)
+	QDEL_IN(chosen_arms, 30)
+	return ..()
```

Comment:
This is a LOT of resistance on top of them now being able to wear EGO armor AND it's not the same when removed. (Becomes 80% resistance on application, removal becomes 33.4% resistance.)